contrib/mesos/pkg/minion/server.go

```diff
@@ -204,6 +205,15 @@ func (ms *MinionServer) launchHyperkubeServer(server string, args *[]string, log
 		log.Infof("wrote %d bytes to %v", written, logFileName)
 	}()
 
+	// use given environment, but add /usr/sbin to the path for the iptables binary used in kube-proxy
```

Comment:
yes, I like the idea to use a flag. It's pretty system dependent and shouldn't be hardcoded.